presto-main/src/main/java/io/prestosql/execution/MemoryRevokingScheduler.java

```diff
@@ -73,7 +79,9 @@ public MemoryRevokingScheduler(
                 requireNonNull(sqlTaskManager, "sqlTaskManager cannot be null")::getAllTasks,
                 requireNonNull(taskManagementExecutor, "taskManagementExecutor cannot be null").getExecutor(),
                 config.getMemoryRevokingThreshold(),
-                config.getMemoryRevokingTarget());
+                config.getMemoryRevokingTarget(),
+                config.isPrioritizeLargerSpiltsMemoryRevoke(),
+                config.getRevocableMemoryLimit());
```

Comment:
this name is not correct; should be something like revokable memory selection threshold

Reply:
fixed